sentinel2download/downloader.py

```diff
@@ -155,14 +149,16 @@ def _blobs_to_load(self, tile_prefix):
     def _download_blob(self, blob):
         save_path = self.get_save_path(blob)
         # check if file exists
-        if not save_path:
-            return True, blob.name
+        if save_path.is_file():
+            logger.info(f"Blob {save_path} exists, skipping download")
```

Comment:
See the comment above about save_path.touch() - this is needed because of https://utility.quantumobile.co/task?id=13749